src/anchorfm-pupeteer/index.js

```diff
@@ -200,6 +187,33 @@ function getSaveDraftOrPublishOrScheduleButtonDescription() {
   };
 }
 
+
+async function clickSelector(page, selector, {visible} = {}) {
+  const options = {};
+  if(visible !== undefined){
+    options.visible = visible;
+  }
+  await page.waitForSelector(selector, options);
+  const element = await page.$(selector);
+  await clickDom(page, element);
+}
+
+async function clickXpath(page, xpath, {timeout} = {}){
+  const options = {};
+  if(timeout !== undefined){
+    options.timeout = timeout;
+  }
+  await page.waitForXPath(xpath, options);
+  const [xpathBtn] = await page.$x(xpath);
+  await clickDom(page,xpathBtn)
+}
+
+async function clickDom(page, domBtn){
```

Comment:
Could  you explain to me what is the benefit of doing this with `page.evaluate` rather than just `elem.click()`?